pipeline/README.MD

```diff
@@ -4,9 +4,11 @@ This directory contains all the necessary code and documentation for the Extract
 
 ## Requirements
 
-To run this script, you will need the following:
+To run this script and containerize into a docker image, you will need the following:
 
-- **Python**: Version 3.8 or higher
+> ⚠️ **Warning**: Make sure you **do not have `asyncio`** or the AWS lambda will not work. ⚠️
```

Comment:
Great use of emojis, very appealing to users. Maybe we can add more like for the secrets, files etc..